.gitignore

```diff
@@ -24,6 +24,7 @@
 # Varies per-deployment.
 /robottelo.properties
 
+
```

Comment:
Is there any reason for this new line? Adding it is out of line with the spacing in the rest of the file and it will make `git blame` a bit messier.